src/core/eme/index.ts

```diff
@@ -171,15 +171,25 @@ function dispose() : void {
 /**
  * Clear EME ressources as the current content stops its playback.
  */
-function clearEME(): Observable<MediaKeys> {
+function clearEME(): Observable<never> {
   return Observable.defer(() => {
+    const observablesArray : Array<Observable<never>> = [];
     if (instanceInfos.$videoElement && shouldUnsetMediaKeys()) {
-      return disposeMediaKeys(instanceInfos.$videoElement)
+      const obs$ = disposeMediaKeys(instanceInfos.$videoElement)
+        .ignoreElements()
         .finally(() => {
           instanceInfos.$videoElement = null;
-        });
+        }) as Observable<never>;
```

Comment:
Ugly :/

In my POV, ignoreElements transforms Observables into Observable<never>.

I might look into the rxjs code to see why that might not be the case

Reply:
:+1:  This is actually scheduled for the next major of RxJS:
https://github.com/ReactiveX/rxjs/pull/2691

:raised_hands: